### DISCUSSION TITLE
Cannot create any groups

### DISCUSSION THREAD

**Discussion by hegelec** [ChurchApps/ChurchAppsSupport#455] [Chums] [answered]
Hello, team! I'm a new user of CHUMS and am just getting familiar with the interface.

As the title suggests, I am having difficulty adding any groups in the interface. Every time I attempt to add a new group in the Groups tab, filling in both the "Category Name" and "Name" fields, the dialog apparently accepts my submission, but then always returns an empty groups list (i.e. "No groups found. Please create a group."

This behaviour is observed using different browsers (Chromium and Firefox) and on different devices, which makes me doubtful something is going wrong on my side.

Nevertheless, please advise me on if/how to provide further information for ID'ing and debugging this issue. Without the "groups" functionality, Chums is pretty handicapped for what our church needs.

Thank you!
Christian

**Comment by Himali-Malvawala** [comment, marked as answer]
@hegelec  Sorry, it took a while but it's fixed now and you should be able to add groups!

**Reply by hegelec** [reply]
So grateful for your reply! Merry Christmas and thank you sincerely for all you do for this amazing project.

**Reply by icadsm** [reply]
> @hegelec Lo sentimos, tardó un poco, pero ya está solucionado y deberías poder agregar grupos.

I have the same problem and in my case it has not been solved.
